rules/defense_evasion/disable_apparmor_attempt.yaml

```diff
@@ -10,14 +10,50 @@
   type: Exec
   category: defense_evasion
   severity: low
-  description: Monitors for the execution of the systemctl utility with the disable flag to disable the AppArmor service. Attackers may 
-    attempt to disable the AppArmor service to evade detection and avoid possible restrictions on their actions and resources.
-  condition: payload.filename == "/usr/bin/systemctl" AND payload.argv CONTAINS "disable" AND payload.argv CONTAINS "apparmor"
+  description: Monitors for attenot to disable the AppArmor service. Attackers may attempt to disable the AppArmor service 
+    to evade detection and avoid possible restrictions on their actions and resources.
+  condition: (
+    (payload.filename STARTS_WITH "/usr/bin/systemctl" AND payload.argv CONTAINS "disable" AND (payload.argv CONTAINS "apparmor" OR payload.argv CONTAINS "apparmor.service")) OR
+    (payload.filename ENDS_WITH "service" AND payload.argv CONTAINS "apparmor" AND payload.argv CONTAINS "stop") OR
+    (payload.filename ENDS_WITH "chkconfig" AND payload.argv CONTAINS "apparmor" AND payload.argv CONTAINS "off")
+    )
```

Comment:
- typo `attenot` in  description
- it should cover `systemctl stop` and `systemctl disable`
- it should cover `service stop` and `service disable` (check `service` syntax)